src/PresentationalComponents/ComplianceTable/ComplianceTable.js

```diff
@@ -31,13 +31,13 @@ const ComplianceTable = (props) => {
   const tableState = useRawTableState();
 
   useDeepCompareEffectNoCheck(() => {
-    if (ENABLE_ASYNC_TABLE_HOOKS) {
+    if (useAPIV2FeatureFlag) {
       console.log('Async Table enabled');
       console.log('TableState', tableState, serialisedTableState);
     }
   }, [tableState, serialisedTableState]);
 
-  return ENABLE_ASYNC_TABLE_HOOKS ? (
+  return useAPIV2FeatureFlag ? (
```

Comment:
Since we have replaced the constant with the feature flag hook call here, let's also cover the undefined case so that we don't unnecessarily render TableToolsTable.

Like this

```suggestion
  return apiV2Enabled === undefined ? render bullseye + spinner : apiV2Enabled === true ? render async table : render TableToolsTable
```

Reply:
@gkarat yep, sounds good, fixed